changelogs/changelog.yaml

```diff
@@ -378,22 +451,118 @@ releases:
+        now looks for error in stdout in the if statement to use -X option. (https://github.com/ansible-collections/ibm_zos_core/pull/641)
+      - zos_copy - Fixed a bug where the module would change the mode for a directory
+        when copying into it the contents of another. (https://github.com/ansible-collections/ibm_zos_core/pull/746)
+      - zos_copy - Fixes a bug where files not encoded in IBM-1047 would trigger an
+        error while computing the record length for a new destination dataset. Issue
+        664. (https://github.com/ansible-collections/ibm_zos_core/pull/725)
+      - zos_copy - Fixes a bug where the code for fixing an issue with newlines in
+        files (issue 599) would use the wrong encoding for normalization. Issue 678.
+        (https://github.com/ansible-collections/ibm_zos_core/pull/725)
+      - zos_job_submit - Fixes the issue when `wait_time_s` was set to 0 that would
+        result in a `type` error that a stack trace would result in the response,
+        issue 670. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when a job encounters a security exception
+        no job log would would result in the response, issue 684. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when a job is configured for a syntax check
+        using TYPRUN=SCAN that it would wait the full duration set by `wait_time_s`
+        to return a response, issue 685. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when a job is configured for a syntax check
+        using TYPRUN=SCAN that no job log would result in the response, issue 685.
+        (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when a job is purged by the system that a
+        stack trace would result in the response, issue 681. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when invalid JCL syntax is submitted that
+        a stack trace would result in the response, issue 623. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue when resources (data sets) identified in
+        JCL did not exist such that a stack trace would result in the response, issue
+        624. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_job_submit - Fixes the issue where the response did not include the job
+        log when a non-zero return code would occur, issue 655. (https://github.com/ansible-collections/ibm_zos_core/pull/683)
+      - zos_operator - fixed incorrect example descriptions and updated the doc to
+        highlight the deprecated option `wait`. (https://github.com/ansible-collections/ibm_zos_core/pull/648)
+      release_summary: 'Release Date: ''2023-4-21''
```

Comment:
I see release dates using 2023-4-21 and 2023-04-21. Not breaking anything but would prefer to stick to one

Reply:
I took care of that, that might have been my doing.

Reply:
Commit [here](https://github.com/ansible-collections/ibm_zos_core/pull/752/commits/1ea7be263f4970ea401881a7fc38be6ceac9387b) should resolve the conversation